src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -392,6 +416,7 @@ export const ShippingFragmentContainer = createFragmentContainer(
         state
         requestedFulfillment {
           __typename
+
```

Comment:
This seems like it will only fetch the `phoneNumber` for shipping orders-- should `phoneNumber` be here instead of nested under that type?

Reply:
Possibly...but I get errors when I move it

Reply:
You should be able to add:
```
... on CommercePickup {
  phoneNumber
}
```

Reply:
```Unknown field 'phoneNumber' on type 'CommercePickup'```